compact/display.xml

```diff
@@ -139,24 +141,32 @@
     <comment>
       Backwards beamline vacuum
     </comment>
-    <vis name="BackwardsBox"      alpha="0.3" ref="AnlGray"       showDaughters="true" visible="true"/>
-    <vis name="BackwardsAir"      alpha="0.7" ref="AnlBlue"       showDaughters="true" visible="true"/>
-    <vis name="BackwardsVac"      alpha="0.1" ref="AnlLight_Gray" showDaughters="true" visible="true"/>
+    <vis name="BackwardsBox"      alpha="0.3" r="102/256" g="102/256" b="102/256"     showDaughters="true" visible="true"/>
+    <vis name="BackwardsVac"      alpha="0.5" r="209/256" g="209/256" b="209/256"     showDaughters="true" visible="true"/>
+    <vis name="BackwardsAir"      alpha="0.5" r="0/256"   g="96/256"  b="156/256"     showDaughters="true" visible="true"/>
+    <comment>
+      Commented versions while bug in DD4Hep overwites alpha value of ref
```

Comment:
Please refer to DD4hep bug and report. We should fix the bug instead of working around it, especially since this is not mission critical and we can wait.